JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -157,5 +157,19 @@
             var propertyValueDump: string = JSON.stringify(object);
             return objectTypeDump + propertyValueDump;
         }
+
+        /**
+         * Returns a boolean indicating whether the browser is IE or not.
+         * @param {string} userAgent - The user agent string for the browser
+         */
+        public static isBrowserIE(userAgent: string): boolean {
+            if (!userAgent) {
+                throw new Error('No user agent provided');
```

Comment:
Makes sense. I was looking at the problem other way round. Thanks for the feedback. I'll change this.